### PR TITLE
Corrected typos and added punctuation marks to sentences missing them.

### PR DESCRIPTION
**Pull request by Teebarh**
I corrected some typographical errors in the documentation and added punctuation marks to sentences where needed.

### CODE REVIEW

File: docs/getting_started/integrating_into_django.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Integrating Wagtail into a Django project
 
-Wagtail provides the `wagtail start` command and project template to get you started with a new Wagtail project as quickly as possible, but it's easy to integrate Wagtail into an existing Django project too.
+Wagtail provides the `wagtail start` command, and project template, to get you started with a new Wagtail project as quickly as possible. However, It is easy to integrate Wagtail into an existing Django project as well.
```
Comment (thibaudcolas) [inline docs/getting_started/integrating_into_django.md:3]: I’m not finding this reads any better to be honest.

```suggestion
Wagtail provides the `wagtail start` command and project template to get you started with a new Wagtail project as quickly as possible, but it's easy to integrate Wagtail into an existing Django project too.
```

File: docs/advanced_topics/accessibility_considerations.md [hunk 2]
```diff
@@ -73,14 +73,14 @@ If this is a concern to you, you can change which tags are used when saving cont
 
 ### TableBlock
 
-The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss they need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
+The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss. They need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
```
Comment (thibaudcolas) [inline docs/advanced_topics/accessibility_considerations.md:76]: The sentence would make no sense with this change included! It’s the need for row and column headers that people will miss.

```suggestion
The [TableBlock](../reference/contrib/table_block) default implementation makes it too easy for end-users to miss they need either row or column headers ([#5989](https://github.com/wagtail/wagtail/issues/5989>)). Make sure to always have either row headers or column headers set.
```

**Review by thibaudcolas — APPROVED** [review]
Thank you @Teebarh. In the future please try to make other contributions than grammar fixes / typos – now that you’ve made your first pull request, there are better ways to demonstrate your skillset!

**Comment by Teebarh** [comment]
> Thank you @Teebarh. In the future please try to make other contributions than grammar fixes / typos – now that you’ve made your first pull request, there are better ways to demonstrate your skillset!

@thibaudcolas, do you have any suggestions sir? I'm so confused on other ways I can contribute to the documentation as I'm completely new to open source contribution😣.

**Pull request merged**